# Name of the cocktail recipe
string cocktail_name

---
# Confirmation of request
bool confirmation
